src/system_simulation/server/simulation_runner.rb

```diff
@@ -10,11 +10,15 @@
 require_relative '../animation_data_sample.rb'
 require_relative './generate_modelica_model.rb'
 
+
 # This class encapsulates the way of how system simulations (physically correct simulations of the dynamic system,
 # including spring oscillations) are run. Right now we use Modelica and compile / simulate a modelica model of our
 # geometry when necessary. This class provides public interfaces for different results of the simulation.
 class SimulationRunner
-  NODE_COORDINATES_FILTER = 'node_[0-9]+.r_0.*'.freeze
+  NODE_RESULT_FILTER = 'node_[0-9]+.r_0.*'.freeze
```

Comment:
Maybe not necessary in this PR, but the ```.``` there is actually quite misleading, because it matches any character, but we want to match only dots.
So I think ```'node_[0-9]+\.r_0\..*'``` better communicates what we really want to know.